docker/electrs/Dockerfile

```diff
@@ -0,0 +1,34 @@
+ARG VERSION=v0.8.6
```

Comment:
Looks like the most recent release is v0.8.9, any reason not to use that instead?